packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -84,7 +83,7 @@ export function TableNG(props: TableNGProps) {
 
       return {
         columnKey,
-        direction: desc ? ('DESC' as const) : ('ASC' as const),
+        direction: desc ? 'DESC' : 'ASC',
```

Comment:
@ashharrison90  not sure what to do here

Reply:
@leeoniya this isn't a type assertion, it's a [const assertion](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions), so you can just revert this change

this is us saying to typescript "do not widen `DESC`/`ASC` to string, because we know it can never be anything other than `DESC`/`ASC`"

if you want to see where type assertions are in a file, you can either:
- temporarily add `/* eslint-enable @typescript-eslint/consistent-type-assertions */` to the top of the file
- add 
  ```ini
  [frontend_dev]
  betterer_eslint_rules = true
  ```
  to your `custom.ini` to see all betterer rules in your editor